src/Symfony/Component/Console/Output/ConsoleOutput.php

```diff
@@ -41,6 +41,12 @@ public function __construct(int $verbosity = self::VERBOSITY_NORMAL, bool $decor
     {
         parent::__construct($this->openOutputStream(), $verbosity, $decorated, $formatter);
 
+        if (null === $formatter) {
+            $this->stderr = new StreamOutput($this->openErrorStream(), $verbosity, $decorated, $formatter);
```

Comment:
I suggest using `null` or omitting the argument rather than passing `$formatter`, to make the code easier to understand.

Reply:
fixed